data/Tag.json

```diff
@@ -68,5 +68,15 @@
     "id": 14,
     "description": "Intended for research only",
     "name": "research"
+  },
+  {
+    "id": 15,
+    "description": "Blocks specific topics/things",
+    "name": "topical"
+  },
+  {
+    "id": 16,
```

Comment:
I wonder if we might include this in `annoyances` ? We want enough tags to be helfpul, but not so many that they are too specific... Thoughts? I guess I'm fine either way, but I think I would tend to lean towards overlays being an annoyance.